plugins/abnormal_security/icon_abnormal_security/util/api.py

```diff
@@ -91,24 +91,12 @@ def generate_filter_params(self, from_date: str = None, to_date: str = None, fil
         if from_date or to_date:
             params = {"filter": filter_key}
             if from_date:
-                params["filter"] = params["filter"] + f" gte {self.parse_date(from_date)}"
+                params["filter"] = params["filter"] + f" gte {from_date}"
```

Comment:
I think that'd be better to have everything in f-strings 😄 
```suggestion
                params["filter"] = f"{params.get('filter', '')} gte {from_date}"
```

Reply:
This has been updated to use f-strings